plugins/module_utils/linode_helper.py

```diff
@@ -110,10 +110,17 @@ def mapping_to_dict(obj: Any) -> Any:
 
 
 def handle_updates(
-    obj: linode_api4.Base, params: dict, mutable_fields: set, register_func: Any
+    obj: linode_api4.Base,
+    params: dict,
+    mutable_fields: set,
+    register_func: Any,
+    ignore_keys: Set[str] = None,
 ) -> Set[str]:
     """Handles updates for a linode_api4 object"""
 
+    if ignore_keys is None:
+        ignore_keys = set()
+
```

Comment:
Although not critical, using `ignore_keys = ignore_keys or set()` might be more consistent with the format below